packages/smooth_app/lib/bottom_sheet_views/product_copy_view.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class CopyToListView extends StatelessWidget {
+  const CopyToListView(
+    this._scrollController, {
```

Comment:
I'm not sure how relevant it is to make the `ScrollController` travel from the caller to here. I'm not even sure why it's used anyway - wouldn't it work by default without an explicit one?